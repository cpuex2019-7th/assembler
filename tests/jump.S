label1:
    addi t0, zero, 1
    j label3
label2:
    li t3, 0x12345678
    addi t1, t1, 2
    j ret
label3:
    addi t1, zero, 3
    j label2
ret:
    add a0, t0, t1
