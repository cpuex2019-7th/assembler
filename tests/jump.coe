memory_initialization_radix=16;
memory_initialization_vector=00100293,00c0006f,00230313,00c0006f,00300313,ff5ff06f,00628533;
radix=16;
