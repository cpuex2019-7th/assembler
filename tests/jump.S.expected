addi t0, zero, 0x1
jal zero, 0xc 
addi t1, zero, 0x2
jal zero, 0xc 
addi t1, zero, 0x3
jal zero, 0xfffffff4 
addi zero, zero, 0x0
